.gitignore

```diff
@@ -4,6 +4,7 @@
 /dist
 /tmp
 /out-tsc
+/coverage
```

Comment:
Added coverage folder. It is getting generated after executing unit tests. It must be ignored from list of file changes